test_scripts/API/AddCommand/005_ATF_AddCommand_UI_AddCommand_WARNINGS.lua

```diff
@@ -0,0 +1,142 @@
+-- SDL -> MOB: AddCommand (result code: WARNINGS, success: true)
+---------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.SDLStoragePath = config.pathToSDL .. "storage/"
+-- ToDo (vvvakulenko): remove after issue "ATF does not stop HB timers by closing session and connection" is resolved
+config.defaultProtocolVersion = 2
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require ('user_modules/shared_testcases/commonSteps')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+
+--[[ Local Variables ]]
+local storagePath = config.SDLStoragePath..config.application1.registerAppInterfaceParams.appID.. "_" .. config.deviceMAC.. "/"
+local ServerAddress = commonFunctions:read_parameter_from_smart_device_link_ini("ServerAddress") 
+
+--[[ General Precondition before ATF start ]]
+commonFunctions:SDLForceStop()
+commonSteps:DeleteLogsFiles()
+commonSteps:DeletePolicyTable()
+testCasesForPolicyTable:precondition_updatePolicy_AllowFunctionInHmiLeves({"BACKGROUND", "FULL", "LIMITED"},"AddCommand")
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_ActivationApp()
+  local request_id = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = self.applications[config.application1.registerAppInterfaceParams.appName]})
+  EXPECT_HMIRESPONSE(request_id)
+  :Do(function(_,data)
+    if (data.result.isSDLAllowed ~= true) then
+      local request_id1 = self.hmiConnection:SendRequest("SDL.GetUserFriendlyMessage", {language = "EN-US", messageCodes = {"DataConsent"}})
+      EXPECT_HMIRESPONSE(request_id1)
+      :Do(function(_,_)   
+        self.hmiConnection:SendNotification("SDL.OnAllowSDLFunctionality", {allowed = true, source = "GUI", device = {id = config.deviceMAC, name = ServerAddress}})
+        EXPECT_HMICALL("BasicCommunication.ActivateApp")
+        :Do(function(_,data1)
+          self.hmiConnection:SendResponse(data1.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+        end)
+      end)
+    end
+  end)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"}) 
+end
+
+commonSteps:PutFile("Precondition_PutFile", "icon.png")
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+local resultCodes = {"SUCCESS", "WARNINGS", "WRONG_LANGUAGE", "RETRY", "SAVED"}
+  for i=1,#resultCodes do
+    Test["TestStep_AddCommand_UI_AddCmd_WARNINGS_and_VR_AddCmd_"..resultCodes[i]] = function(self)
+    local cid = self.mobileSession:SendRPC("AddCommand",
```

Comment:
@vvvakulenko, after first successful AddCommand, SDL will reject next AddCommand, because of:
 cmdID = 11 - INVALID_ID,
menuName ="Commandpositive" - DUPLICATE_NAME
vrCommands = {....} - DUPLICATE_NAME

So you can use counter 'i' in the request. As example:
cmdID = i,
menuName ="Commandpositive" .. tostring(i)
"VRCommandonepositive" .. tostring(i),
"VRCommandonepositivedouble" .. tostring(i)

Please don't forget also to updated expected results according to data sent in mobile request.